mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -362,8 +362,13 @@ def evaluate(self,
                 results, resfile_path, metrics)
             print_log('Evaluate CLEAR MOT results.', logger=logger)
             distth = 1 - track_iou_thr
-
             accs = []
+            # support loading data from ceph
+            client = self.file_client.client
+            local_dir = tempfile.TemporaryDirectory()
+            # avoid using root paths
+            tmp_prefix = '/mmtracking' + os.sep + self.img_prefix.split(
```

Comment:
Can we use `osp.join()` instead of this joining string?

Reply:
ok